db/migrate/20190320232413_update_null_check.rb

```diff
@@ -0,0 +1,6 @@
+class UpdateNullCheck < ActiveRecord::Migration[5.2]
+  def change
+    change_column_null :delay_henka_scheduled_changes, :changeable_id, true
+    change_column_null :delay_henka_scheduled_changes, :attribute_name, true
```

Comment:
These losening constraints make me feel that we should just create a new table for create actions, which is probably a good idea since I can see us storing more data in the future that are uniq to create